src/families/hedera/js-transaction.ts

```diff
@@ -0,0 +1,46 @@
+import BigNumber from "bignumber.js";
+import type { Account } from "../../types";
+import type { Transaction } from "./types";
+
+/**
+ * Creates an empty transaction.
+ *
+ * @returns {Transaction}
+ */
+export function createTransaction(_account: Account): Transaction {
+  return {
+    family: "hedera",
+    amount: new BigNumber(0),
+    recipient: "",
+    useAllAmount: false,
+  };
+}
+
+/**
+ * Update a base property of the transaction.
+ *
+ * @returns  {Transaction}
+ */
+export function updateTransaction(
+  transaction: Transaction,
+  patch: Partial<Transaction>
+): Transaction {
+  return { ...transaction, ...patch } as unknown as Transaction;
```

Comment:
I think you don't need this double cast?

Reply:
👍🏻